testproject/Assets/Scenes/ZooSam.unity

```diff
@@ -1192,41 +1192,15 @@ GameObject:
   - component: {fileID: 108150028}
```

Comment:
Would it be possible to leave scene changes separated from the NetworkTransform PR? I don't see how but this seem to trigger some strange behaviour with my setup and prevents me from testing this PR